services/httpd/handler.go

```diff
@@ -747,10 +753,14 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, user meta.U
 		}
 
 		// if requested, convert result timestamps to epoch
-		if epoch != "" {
+		if epoch != "" && timeFormat == "epoch" {
 			convertToEpoch(r, epoch)
 		}
 
+		if timeFormat == "rfc3339" {
+			convertToRfc3339(r)
+		}
```

Comment:
Are these two mutually exclusive? If so, it should probably be an if-then-else.

It seems like setting `time_format=rfc3339` and `epoch=foo` should give the user an error to avoid support calls of people trying to combine them to get RFC3339 with second precision or such.